modules/gov/expected_modules.go

```diff
@@ -26,6 +26,7 @@ type SlashingModule interface {
 
 type StakingModule interface {
 	GetStakingPool(height int64) (*types.Pool, error)
+	GetStakingPoolSnapshot(height int64) (*types.PoolSnapshot, error)
```

Comment:
On testnet, I saw there were swaps between Pool to PoolSnapshot, this keeps both.

Reply:
Both are kept on the testnet and upstream.